crowdsourcing/dialogues/multi_party_chat/webapp/src/utils.js

```diff
@@ -0,0 +1,168 @@
+// TODO: pass the list of stop words and offensive words from ParlAI instead of hard-coding them
+const STOPWORDS = ['', "\'\'", "\'d", "\'ll", "\'m", "\'re", "\'s", "\'ve", '*',
+    ',', '--', '.', '?', '\`\`', 'a', 'about', 'above', 'after',
+    'again', 'against', 'ain', 'all', 'also', 'am', 'an', 'and',
+    'any', 'are', 'aren', 'as', 'at', 'be', 'because', 'been',
+    'before', 'being', 'below', 'between', 'both', 'but', 'by',
+    'can', 'couldn', 'd', 'did', 'didn', 'do', 'does', 'doesn',
+    'doing', 'don', 'down', 'during', 'each', 'few', 'for', 'from',
+    'further', 'had', 'hadn', 'has', 'hasn', 'have', 'haven',
+    'having', 'he', 'her', 'here', 'hers', 'herself', 'him',
+    'himself', 'his', 'how', 'i', 'if', 'in', 'into', 'is', 'isn',
+    'it', 'its', 'itself', 'just', 'know', 'll', 'm', 'ma', 'me',
+    'mightn', 'more', 'most', 'mustn', 'my', 'myself', "n't",
+    'needn', 'no', 'nor', 'not', 'now', 'o', 'of', 'off', 'on',
+    'once', 'only', 'or', 'other', 'our', 'ours', 'ourselves',
+    'out', 'over', 'own', 'people', 're', 'really', 's', 'same',
+    'see', 'shan', 'she', 'should', 'shouldn', 'so', 'some',
+    'such', 't', 'than', 'that', 'the', 'their', 'theirs', 'them',
+    'themselves', 'then', 'there', 'these', 'they', 'this',
+    'those', 'through', 'to', 'too', 'under', 'until', 'up', 've',
+    'very', 'want', 'was', 'wasn', 'we', 'were', 'weren', 'what',
+    'when', 'where', 'which', 'while', 'who', 'whom', 'why', 'will',
+    'with', 'won', 'wouldn', 'y', 'you', 'your', 'yours',
+    'yourself', 'yourselves'];
+
+const OFFENSIVE_WORDS = ['2g1c', '2 girls 1 cup', 'acrotomophilia',
```

Comment:
Have we just been storing and re-copying this list? I would've expected it to be somewhere backend. (not to be changed right now though!)

Reply:
I agree, this was implemented a bit hacky here. I will merge it like this for now, but we can have it pulling from a static page in the future (maybe one of our own Github files).